src/config/constants.js

```diff
@@ -96,3 +96,6 @@ export const MODAL_RECEIVE = 'MODAL_RECEIVE'
 export const MODAL_SEND = 'MODAL_SEND'
 export const MODAL_SETTINGS_ACCOUNT = 'MODAL_SETTINGS_ACCOUNT'
 export const MODAL_RELEASES_NOTES = 'MODAL_RELEASES_NOTES'
+
+export const MODAL_DISCLAIMER = 'MODAL_DISCLAIMER'
+export const MODAL_DISCLAIMER_DELAY = 1 * 1000
```

Comment:
keep the delay with the constant delays ;)

Reply:
Ah yes oops